ota-plus-web/app/reactapp-new/webpack.config.js

```diff
@@ -20,7 +20,10 @@ module.exports = {
   },
   devtool: 'source-map',
   devServer: {
-    contentBase: resolve(__dirname, 'src')
+    contentBase: resolve(__dirname, 'src'),
+    https: true,
+    disableHostCheck: true,
```

Comment:
Out of curiosity: what does HostCheck actually checks and why we don't need it?

Reply:
It checks origin header, IIRC. We don't need it bcs it will be used in development setup where html is rendered on a server and js is loaded from localhost

Reply:
And how do we distinguish between dev and production setups? Ability load arbitrary js from localhost seems highly undesirable in production.

Reply:
webpack only runs in development